db/migrate/20151028005733_generate_tokens_for_existing_users.rb

```diff
@@ -0,0 +1,7 @@
+class GenerateTokensForExistingUsers < ActiveRecord::Migration
+  def change
+    User.all.each do |user|
```

Comment:
Anytime you are iterating through `all` of a type of object, you should use `find_each`. That will process in batches of 1000, hopefully preventing your machine from running out of memory.